modules/gateway/gateway.go

```diff
@@ -75,18 +75,20 @@ func (g *Gateway) RelayBlock(b consensus.Block) (err error) {
 		return errors.New("block added, but it does not extend the state height.")
 	}
 
-	g.broadcast("RelayBlock", b, nil)
+	go g.threadedBroadcast("RelayBlock", b, nil)
 	return
 }
 
 // RelayTransaction relays a transaction, both locally and to the network.
 func (g *Gateway) RelayTransaction(t consensus.Transaction) (err error) {
-	g.broadcast("AcceptTransaction", t, nil)
```

Comment:
Convention is that all exported functions start with a

```
lock 
defer unlock
```,

 can you add a single comment:
```
// no locking necessary
```